egs/wsj/s5/steps/nnet3/lstm/decode.sh

```diff
@@ -131,7 +135,8 @@ fi
 
 if [ ! -z "$online_ivector_dir" ]; then
   ivector_period=$(cat $online_ivector_dir/ivector_period) || exit 1;
-  ivector_opts="--online-ivectors=scp:$online_ivector_dir/ivector_online.scp --online-ivector_period=$ivector_period"
+  [ $ivector_interval -gt $frames_per_chunk ] && echo "Error ivector_interval should not be greater than frames_per_chunk." && exit 1
+  ivector_opts="--online-ivectors=scp:$online_ivector_dir/ivector_online.scp --online-ivector_period=$ivector_period --ivector-interval=$ivector_interval"
```

Comment:
Please specify the difference between online-ivector-period and ivector-interval in the comments.